data/txHandler.go

```diff
@@ -143,17 +188,17 @@ func (handler *TxHandler) backlogWorker() {
 		select {
 		case wi, ok := <-handler.backlogQueue:
 			if !ok {
+				// this is never happening since handler.backlogQueue is never closed
 				return
 			}
 			if handler.checkAlreadyCommitted(wi) {
 				continue
 			}
-
-			// enqueue the task to the verification pool.
-			handler.txVerificationPool.EnqueueBacklog(handler.ctx, handler.asyncVerifySignature, wi, nil)
+			handler.streamVerifierChan <- verify.UnverifiedElement{TxnGroup: wi.unverifiedTxGroup, Context: wi}
 
 		case wi, ok := <-handler.postVerificationQueue:
 			if !ok {
+				// this is never happening since handler.postVerificationQueue is never closed
```

Comment:
would it make sense to remove this code then, or add some error logging in case this code is exercised by surprise?